cvat/apps/engine/ddln/sequences.py

```diff
@@ -30,11 +30,23 @@ def group(sequences, chunk_size):
     return result
 
 
-def distribute(chunks, assignees):
+def distribute(chunks, assignees, times_assigned=1):
     """Distribute chunks between assignees
 
-    Each assignee gets no more than 1 chunk,
+    If times assigned is equal to 1, each assignee gets no more than 1 chunk,
```

Comment:
this function encapsulates two different logics for case of `times_assigned=1` or `>1`
better make two explicit functions